lib/vmpooler/api.rb

```diff
@@ -168,6 +168,45 @@ def hostname_shorten hostname
           JSON.pretty_generate(result)
         end
 
+        get '/status/?' do
+          content_type :json
+
+          result = {}
+          clone_times = []
```

Comment:
I'd say "not at this point", no... perhaps if our monitoring evolves in the future to need more granular clone-time judgement, but as I envision it currently... no.